src/components/atoms/Avatar/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useMedia } from "react-use";
+
+import { metricsSizes, styled, useTheme } from "@reearth/theme";
+
+import Text from "../Text";
+
+export type Props = {
+  size?: number;
+  avatar?: string;
+  color?: string;
+  userName?: string;
+  innerText?: string;
+};
+
+const Avatar: React.FC<Props> = ({ size = 24, avatar, color, userName, innerText }) => {
+  const theme = useTheme();
+  const isSmallWindow = useMedia("(max-width: 1024px)");
+  return (
```

Comment:
This is maybe more of a preference, but if there is ANYTHING in the component other than the return I usually try to make a space between the return and the last variable or function.